src/NServiceBus.RavenDB/Outbox/OutboxPersister.cs

```diff
@@ -5,17 +5,20 @@
     using System.Threading.Tasks;
     using NServiceBus.Extensibility;
     using NServiceBus.Outbox;
+    using NServiceBus.Pipeline;
     using NServiceBus.RavenDB.Outbox;
     using Raven.Client;
     using TransportOperation = NServiceBus.Outbox.TransportOperation;
 
     class OutboxPersister : IOutboxStorage
     {
-        public OutboxPersister(IDocumentStore documentStore, string endpointName)
+        public OutboxPersister(IDocumentStore documentStore, string endpointName, IOpenRavenSessionsInPipeline sessionCreator = null)
```

Comment:
Are we defaulting the argument to `null` so that all other tests don't need to be changed?

Reply:
Yeah, there are 8 other references in tests.

Reply:
I'd rather change all the tests instead of making production code accomodate tests needs. What do you think?